### PR TITLE
[12.0[IMP] contract add commercial partner

### PR DESCRIPTION
**Pull request by hparfr**
Add commercial_partner_id to contract.

Replaces #361 
Depends of #207 

- [x] https://github.com/OCA/contract/pull/207

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Something seems not correct looking commit history...

**Comment by pedrobaeza** [comment]
OK, now I see that this contains commit history from #207

**Comment by hparfr** [comment]
207 is merged. I did some git merges to keep only the changes without any force.

So the history is currently messy,  but the diff change is minimal. It should work well with a rebase on merge.

File: contract/views/contract.xml [hunk 1]
```diff
@@ -189,6 +190,9 @@
                         domain="[('date_end', '&lt;', context_today().strftime('%Y-%m-%d')), ('recurring_next_date', '=', False)]"
                 />
                 <field name="partner_id"/>
+                <field name="commercial_partner_id"
+                        string="Commerical Entity"
```
Comment (sbidoul) [inline contract/views/contract.xml]: A spelling issue here. The string attribute could be removed.

**Review by sbidoul — COMMENTED** [review]
Looks good at first sight.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
No, it won't work. Please fix the commit history starting from current branch and cherry-picking your commit, or redo the diff, but this can't be merged as is.

**Comment by hparfr** [comment]
@pedrobaeza done as requested.

File: contract/views/contract.xml (hunk 1 above)
Comment (hparfr) [reply contract/views/contract.xml]: Good catch

**Comment by pedrobaeza** [comment]
Thanks, can you please squash both commits together (our bot doesn't still do this)?

**Comment by hparfr** [comment]
@pedrobaeza, it's now squashed in one commit

**Review by pedrobaeza — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**